### PR TITLE
docs(self-hosted): move readme contents

### PR DESCRIPTION
**Pull request by aldy505**
## DESCRIBE YOUR PR
Dupe of https://github.com/getsentry/develop/pull/1339

Closes https://github.com/getsentry/self-hosted/issues/1996

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Comment by aldy505** [comment]
Any feedbacks from devinfra team? @hubertdeng123 @joshuarli 

P.S. I also have a few PRs for updating self-hosted related docs. See https://github.com/getsentry/sentry-docs/pulls?q=is%3Apr+self-hosted+is%3Aopen+author%3Aaldy505

**Comment by aldy505** [comment]
Another ping @joshuarli @hubertdeng123

**Comment by aldy505** [comment]
another ping @hubertdeng123

File: develop-docs/self-hosted/index.mdx [hunk 1]
```diff
@@ -25,26 +25,26 @@ sudo ./install.sh
...
 
-Depending on your usage, required system resource to run Sentry may varies. Minimum system specification for running Sentry looks like this:
-
-- 2 CPU cores
-- 4 GB RAM
-
-Having a machine that have lower system specification than that will fail the installation script.
-
-Current recommended system specification for running Sentry is:
+Depending on your usage, required system resource to run Sentry may varies. Current recommended system specification for running Sentry is:
```
Comment (hubertdeng123) [inline develop-docs/self-hosted/index.mdx:28]: We now have these are system minimum requirements, so should update this from recommended -> required

File: develop-docs/self-hosted/index.mdx [hunk 2]
```diff
@@ -25,26 +25,26 @@ sudo ./install.sh
...
 
 - 4 CPU cores
 - 16 GB RAM
 - 20 GB free disk storage space
 
-Depending on your traffic volume, you may want to increase your system specification to handle increased load. If increasing the disk storage space isn't possible, you can migrate your storage to use external storage such as AWS S3 or Google Cloud Storage (GCS). Decreasing your `SENTRY_RETENTION_DAYS` environment variable to lower numbers will save some storage space from being full, at the cost of having shorter data retention period.
+We'd recommend using a swapfile if you are using a machine with less than 16 GB of RAM. Depending on your traffic volume, you may want to increase your system specification to handle increased load. If increasing the disk storage space isn't possible, you can migrate your storage to use external storage such as AWS S3 or Google Cloud Storage (GCS). Decreasing your `SENTRY_RETENTION_DAYS` environment variable to lower numbers will save some storage space from being full, at the cost of having shorter data retention period. See [Event Retention](#event-retention) section below.
```
Comment (hubertdeng123) [inline develop-docs/self-hosted/index.mdx:34]: We don't want to directly recommend people to use a machine with less than 16 GB of RAM

File: develop-docs/self-hosted/index.mdx [hunk 3]
```diff
@@ -32,13 +32,20 @@ These are the minimum requirements:
 - 16 GB RAM
 - 20 GB Free Disk Space
 
-Depending on your traffic volume, you may want to increase your system specification to handle increased load. If increasing the disk storage space isn't possible, you can migrate your storage to use external storage such as AWS S3 or Google Cloud Storage (GCS). Decreasing your `SENTRY_RETENTION_DAYS` environment variable to lower numbers will save some storage space from being full, at the cost of having shorter data retention period.
+We'd recommend using a swapfile if you are using a machine with less than 16 GB of RAM. Depending on your traffic volume, you may want to increase your system specification to handle increased load. If increasing the disk storage space isn't possible, you can migrate your storage to use external storage such as AWS S3 or Google Cloud Storage (GCS). Decreasing your `SENTRY_RETENTION_DAYS` environment variable to lower numbers will save some storage space from being full, at the cost of having shorter data retention period. See [Event Retention](#event-retention) section below.
```
Comment (aldy505) [inline develop-docs/self-hosted/index.mdx:35]:
```suggestion
We'd recommend using a swapfile whenever applicable. Depending on your traffic volume, you may want to increase your system specification to handle increased load. If increasing the disk storage space isn't possible, you can migrate your storage to use external storage such as AWS S3 or Google Cloud Storage (GCS). Decreasing your `SENTRY_RETENTION_DAYS` environment variable to lower numbers will save some storage space from being full, at the cost of having shorter data retention period. See [Event Retention](#event-retention) section below.
```

**Comment by aldy505** [comment]
ping @hubertdeng123 @joshuarli

**Review by joshuarli — APPROVED** [review]
Looks good to me, very well done.

**Pull request merged**